src/main/target/UAVPNG030MINI/target.h

```diff
@@ -0,0 +1,157 @@
+#define MPU_INT_EXTI             PE0
+#define USE_MPU_DATA_READY_SIGNAL
+
+#define MAG
+#define USE_MAG_HMC5883
+#define MAG_HMC5883_ALIGN        CW90_DEG
+#define MAG_I2C_INSTANCE         I2CDEV_2
+#define HMC5883_I2C_INSTANCE     I2CDEV_2
+#define MAG_INT_EXTI             PE12
+#define USE_MAG_DATA_READY_SIGNAL
+
+#define BARO
+#define USE_BARO_MS5611
+#define USE_BARO_SPI_MS5611
+#define MS5611_CS_PIN           PE1
+#define MS5611_SPI_INSTANCE     SPI1
+
+#if 0 // TODO: Enable SDCard and blackbox logging
+#define ENABLE_BLACKBOX_LOGGING_ON_SDCARD_BY_DEFAULT
+#define USE_SDCARD
+#define USE_SDCARD_SPI2
+#define SDCARD_DETECT_INVERTED
+#define SDCARD_DETECT_PIN               PE2
+#define SDCARD_SPI_INSTANCE             SPI2
+#define SDCARD_SPI_CS_PIN               SPI2_NSS_PIN
+// SPI2 is on the APB1 bus whose clock runs at 84MHz. Divide to under 400kHz for init:
+#define SDCARD_SPI_INITIALIZATION_CLOCK_DIVIDER 256 // 328kHz
+// Divide to under 25MHz for normal operation:
+#define SDCARD_SPI_FULL_SPEED_CLOCK_DIVIDER 4 // 21MHz
+
+#define SDCARD_DMA_CHANNEL_TX                   DMA1_Stream4
+#define SDCARD_DMA_CHANNEL_TX_COMPLETE_FLAG     DMA_FLAG_TCIF4
+#define SDCARD_DMA_CLK                          RCC_AHB1Periph_DMA1
+#define SDCARD_DMA_CHANNEL                      DMA_Channel_0
+#elif defined(LUXF4OSD)
+#define ENABLE_BLACKBOX_LOGGING_ON_SPIFLASH_BY_DEFAULT
+#define M25P16_CS_PIN           PB12
+#define M25P16_SPI_INSTANCE     SPI2
+#define USE_FLASHFS
+#define USE_FLASH_M25P16
+#endif
+
+#define USE_VCP
+//#define VBUS_SENSING_PIN PC5
+
+#define USE_UART1
+#define UART1_RX_PIN            PB7
+#define UART1_TX_PIN            PB6
+//#define UART1_AHB1_PERIPHERALS  RCC_AHB1Periph_DMA2
+
+#define USE_UART2
+#define UART2_RX_PIN            PD6
+#define UART2_TX_PIN            PD5
+
+#define USE_UART3
+#define UART3_RX_PIN            PD9
+#define UART3_TX_PIN            PD8
+
+#define USE_UART6
+#define UART6_RX_PIN            PC6
+#define UART6_TX_PIN            PC7
+
+#define SERIAL_PORT_COUNT       5 //VCP, USART1, USART2, USART3, USART6
+
+// TODO
+#define USE_ESCSERIAL
+#define ESCSERIAL_TIMER_TX_HARDWARE 0 // PWM 1
+
+#define USE_SPI
+#define USE_SPI_DEVICE_1
+#define SPI1_NSS_PIN            PA4
+#define SPI1_SCK_PIN            PA5
+#define SPI1_MISO_PIN           PA6
+#define SPI1_MOSI_PIN           PA7
+
+/// I2C Configuration
+#define USE_I2C
+#define USE_I2C_PULLUP
+#define USE_I2C_DEVICE_1
+#define I2C1_SCL                PB8
+#define I2C1_SDA                PB9
+
+#define USE_I2C_DEVICE_2
+#define I2C2_SCL                PB10
+#define I2C2_SDA                PB11
+
+#define I2C_DEVICE              (I2CDEV_2)
+
+#define USE_ADC
+#define VBAT_ADC_PIN            PC1
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+
+//#define TRANSPONDER
+
+#define DEFAULT_RX_FEATURE      FEATURE_RX_SERIAL
+#define SERIALRX_PROVIDER       SERIALRX_SBUS
+#define SERIALRX_UART           SERIAL_PORT_USART1
+
+#define USE_SERIALRX_SBUS
```

Comment:
These do not need to be defined explicitly, as they are default on F4.

Reply:
I've removed them.